src/Entity/Service.php

```diff
@@ -150,6 +150,39 @@ protected function getErrandServiceIndex(ErrandService $errand_service) {
   public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
     $fields = parent::baseFieldDefinitions($entity_type);
 
+    $fields['name_synonyms'] = BaseFieldDefinition::create('string_long')
```

Comment:
This field should probably have an unlimited cardinality, then separate the `name_synonyms` field by comma. See Phone field (Unit entity/migration) for an example how to implement this.

Reply:
@tuutti yes, logically make sense to be multiple. but if we do the name synonyms multiple, then won't be able to use combine search field (one requirements was to search in name synonyms) we would have to maybe do aggregation on view which I think would overcomplicate things for this feature.